hotkey.js

```diff
@@ -100,7 +100,13 @@ export function hotkeyKeyUX(transformers = []) {
         event.target.isContentEditable ||
         event.target.tagName === 'TEXTAREA' ||
         (event.target.tagName === 'INPUT' && !IGNORE_INPUTS[event.target.type])
-      let insideFocusGroup = event.target.role === 'menuitem'
+        
+      let insideFocusGroup =
+        event.target.role === 'menuitem' ||
+        event.target.role === 'button' ||
```

Comment:
Why we added all of these?

Should we also add `focusgroup`?

Reply:
When focus is on an element within a focusgroup, hotkeys are restricted to that group only. 
The filtering of elements follows the same logic as implemented in the `getItems`

```js 
function getItems(group) {
  if (group.hasAttribute('focusgroup')) {
    let items = [...group.querySelectorAll('*:not([focusgroup="none"])')]
    return items.filter(item => {
      return (
        item.role === 'button' ||
        item.type === 'button' ||
        item.role === 'checkbox' ||
        item.type === 'checkbox'
      )
    })
  }
}
```

Reply:
We should disable one-letter hotkeys only for cases where we will have a conflict.

In `menuitem` we have a search to find next element. This is why we disable only one-letter hot keys, because they will conflict with this search.

As I understand `focusgroup` has no search, so there will be no conflicts.

Reply:
Looks like you're right, there's nothing about search in the spec. In that case, I'll remove all search-related functionality from the polyfill and update the API to `focusGroupPolyfill()` without the `FocusGroupKeyUXOptions`.